.github/workflows/windows_release.yml

```diff
@@ -0,0 +1,134 @@
+name: Windows Releases
```

Comment:
This is the actual workflow that builds Windows release packages for us now. Key points:

- Runs on Windows server 2019
- Builds 32-bit and 64-bit versions
- Both 32 and 64 are linked against Python
- Is run only on tags, not regular commits
- Builds EnergyPlus against Python 3.7
- Builds and packages up the expected installer types: ZIP and IFW